src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -679,6 +679,17 @@ protected override void MethodChecks(BindingDiagnosticBag diagnostics)
 
             CheckModifiers(MethodKind == MethodKind.ExplicitInterfaceImplementation, _location, diagnostics);
         }
+
+        internal override void AfterAddingTypeMembersChecks(ConversionsBase conversions, BindingDiagnosticBag diagnostics)
+        {
+            base.AfterAddingTypeMembersChecks(conversions, diagnostics);
+
+            if (this.ReturnType?.IsErrorType() == true && this.ReturnType.Name == "partial")
+            {
+                var available = MessageID.IDS_FeaturePartialEventsAndConstructors.CheckFeatureAvailability(diagnostics, DeclaringCompilation, ReturnTypeLocation);
```

Comment:
>MessageID.IDS_FeaturePartialEventsAndConstructors.CheckFeatureAvailability(diagnostics, DeclaringCompilation, ReturnTypeLocation) [](http://example.com/codeflow?start=32&length=129)

Does this check result in a language version error for the following when compiling with earlier language versions? If so, that seems incorrect.
```csharp
class C
{
    @partial F() => null;
}
```

Reply:
Thanks, fixed.